thunder.install

```diff
@@ -89,3 +89,28 @@ function _thunder_mark_update_checklist($update_id, $successful, UpdateLogger $u
     // If service is not available, we will just ignore it.
   }
 }
+
+/**
+ * Installs the autosave_form module.
+ */
+function thunder_update_8301() {
+
+  \Drupal::service('module_installer')->install([
```

Comment:
We can enable modules within the updates yml files. See https://github.com/BurdaMagazinOrg/thunder-distribution/blob/8.x-2.x/config/update/v1_1__content_lock.yml

Reply:
Or https://github.com/BurdaMagazinOrg/thunder-distribution/blob/8.x-2.x/config/update/thunder__thunder_update_8118.yml